src/pages/lookout-application.js

```diff
@@ -0,0 +1,306 @@
+import React, { useState } from "react";
+import { StaticQuery, graphql } from "gatsby";
+
+import Widgets from "../components/reporter/widgets/Widgets";
+import useUser from "../hooks/UserContext";
+import { useMoralis } from "react-moralis";
+
+import ProtectedPage from "../components/ProtectedPage";
+
+const config = {
+  labelAll: "candidate",
+  fields: [
+    {
+      name: "bio",
+      label: "Tell us about yourself",
+      helpText: "Short bio/intro and summary of relevant experience",
+      widget: "textarea",
+      required: true,
+    },
+    {
+      name: "experience",
+      label: "Do you have experience evaluating other people's work?",
+      helpText:
+        "(e.g. instructor, teaching assistant, team manager, etc.) If so, please tell us a little about it.",
+      widget: "textarea",
+      required: false,
+    },
+    {
+      name: "questions",
+      label: "What questions do you have about the Lookout Guidelines?",
+      helpText: (
+        <a
+          href="https://www.notion.so/Pre-sort-Lookout-Guidelines-bead139f7ec9469197c7d46f759d89d5"
```

Comment:
Can we change this URL? I just realized that that URL isn't public facing, but this one is: 

https://code4rena.notion.site/Pre-sort-Lookout-Guidelines-bead139f7ec9469197c7d46f759d89d5